app/build.gradle

```diff
@@ -122,26 +122,26 @@ dependencies {
     implementation 'androidx.room:room-runtime:2.0.0'
     kapt 'androidx.room:room-compiler:2.0.0'
     implementation 'androidx.room:room-rxjava2:2.0.0'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
I think `kotlin-stdlib-jdk7` is the right dependency

Reply:
oops. yes. fixed :)